src/server/pbsd_init.c

```diff
@@ -1709,11 +1708,22 @@ pbsd_init_job(job *pjob, int type)
 			(pjob->ji_qs.ji_un.ji_exect.ji_momaddr != 0)) {
 
 			if (pjob->ji_wattr[(int)JOB_ATR_exec_host].at_flags & ATR_VFLAG_SET) {
-				pjob->ji_qs.ji_un.ji_exect.ji_momaddr =
+				pbs_net_t new_momaddr;
+				unsigned int new_momport;
+
+				new_momaddr =
 					get_addr_of_nodebyname(
 					pjob->ji_wattr[(int)JOB_ATR_exec_host].
-					at_val.at_str, &d);
-				pjob->ji_qs.ji_un.ji_exect.ji_momport = d;
+					at_val.at_str, &new_momport);
+
+				if (new_momaddr != 0) {
+					pjob->ji_qs.ji_un.ji_exect.ji_momaddr = new_momaddr;
+					pjob->ji_qs.ji_un.ji_exect.ji_momport = new_momport;
+				} else {
+					log_event(PBSEVENT_DEBUG3, PBS_EVENTCLASS_JOB,
```

Comment:
I am just wondering if info would be a better log level here.

Reply:
@prakashcv13 Thanks.

LOG_INFO sounds good. I agree. Should we change the event type too?

I think yes. This log message will not spam the log. The occurrence of the message is rare, and what is more important the message can occur only once -  changing debug level and restarting service will not invoke the message reliably. I would say PBSEVENT_DEBUG is suitable as a type. Or maybe higher?

Reply:
I agree @vchlum

Reply:
Done, please see @prakashcv13 .